torchvision/transforms/transforms.py

```diff
@@ -1953,3 +1953,126 @@ def forward(self, img):
 
     def __repr__(self):
         return self.__class__.__name__ + '(p={})'.format(self.p)
+
+
+class RandomMixupCutmix(torch.nn.Module):
```

Comment:
High-level comment: should we split this function in two, doing `MixUp` and `CutMix`?

Reply:
It's an option but there are caveats:
1. Mixup and Cutmix are not compatible and must not be combined one after the other. 
2. It's very common to switch between them randomly and many recipes use this. Separating them would require us to find another way to randomize between them. 
2. ClassyVision's implementation has them together so ours wont be a replacement after its deprecation.

Reply:
As discussed with @fmassa offline, we decided to separate these two and introduce another transform that switches between the 2 randomly.